.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-16.04
+
+    steps:
+    - uses: actions/checkout@v2
+      
+    - name: Install dependencies
+      run: sudo apt install yarn
+      
+    - name: Install palava-web
+      run: |
+        yarn
+        yarn build
+
+    - name: Lint palava-web
+      run: yarn lint
+    
+    - name: Audit palava-web
+      run: yarn audit
```

Comment:
@erikzenker One more thing:
- I think `$ yarn audit --groups dependencies` would be a sensible setting here, since almost all of the security warnings in dev gems are not relevant for the actual application (and create a lot of noise)

Reply:
Will add it